roles/filetree_create/templates/current_applications.j2

```diff
@@ -3,13 +3,25 @@
 controller_applications:
 {% endif %}
   - name: "{{ current_applications_asset_value.name }}"
-    description: "{{ current_applications_asset_value.description }}"
+    description: "{{ template_overrides_resources.application[current_applications_asset_value.name].description
+                     | default(template_overrides_global.application.description)
```

Comment:
Makes it sense setting all the exported `applications` with the same (`template_overrides_global`) description? **This applies to the description of all the other modified objects.**

Reply:
This should be resolved.

Reply:
Sometimes it would be good to set global description, because maybe someone wants to use the description as some kind of indicator.